src/utils/randomString.ts

```diff
@@ -0,0 +1,17 @@
+/*
+ * SPDX-FileCopyrightText: 2021 The HedgeDoc developers (see AUTHORS file)
+ *
+ * SPDX-License-Identifier: AGPL-3.0-only
+ */
+import { randomBytes } from 'crypto';
+
+/**
+ * Generate a random string of defined length in bytes.
+ * @param {number} length - the length in bytes.
+ */
+export function randomString(length: number): Buffer {
+  if (length <= 0) {
+    throw new Error('randomString cannot have a length < 1');
+  }
+  return randomBytes(length);
```

Comment:
Uhm, is this function really needed? All it does is checking that the expected string length to be bigger than 0. Is there any place where we let users control this length parameter? On my review I was only able to spot hardcoded settings. So the if clause is never reached and the function boils down to a fassade.

Reply:
> Is there any place where we let users control this length parameter?

I'm leaning towards no, so I think this can be removed…

Reply:
done